Network/Curl/Opts.hs

```diff
@@ -281,6 +282,11 @@ type OpenSocketFunction
   -> Ptr () -- ^ The struct describing the curl_sockaddr struct
   -> IO CInt -- ^ The opened socket or CURL_SOCKET_BAD
 
+type CloseSocketFunction
+   = Ptr ()  -- ^ Pointer to clientp data set by OpenSocketData
+  -> CInt    -- ^ The curl socket type
+  -> IO CInt -- ^ Whether closing was successful
```

Comment:
What are the values when closing is successful, 0 and 1?

Reply:
Precisely, in typical c style 0 if no error else 1.

Reply:
Could you maybe document that, not everyone might be familiar with that convention.